### ISSUE TITLE
eslint-plugin: Is not compatible with eslint 8.x

### ISSUE THREAD

**Issue by yethee** [microsoft/griffel#505]
I faced with issue when update the plugin to version `1.6.0`:

```
Oops! Something went wrong! :(

ESLint: 8.56.0

Error [ERR_REQUIRE_ESM]: require() of ES Module <redacted>\node_modules\@griffel\eslint-plugin\src\index.js from <redacted>\node_modules\@eslint\eslintrc\dist\eslintrc.cjs not supported.
Instead change the require of index.js in <redacted>\@eslint\eslintrc\dist\eslintrc.cjs to a dynamic import() which is available in all CommonJS modules.
```

**Dependencies:**
```json
{
  "devDependencies": {
    "@griffel/eslint-plugin": "^1.6.0",
    "@typescript-eslint/eslint-plugin": "^6.9.1",
    "@typescript-eslint/parser": "^6.9.1",
    "eslint": "^8.52.0",
  }
}
```

**Affected versions:** `1.5.2, 1.6.0`

There no issue when using the plugin version `1.5.1` or lower.

**Comment by jvince** [comment]
Same issue here. ESLint version used `8.56.0`. As mentioned by @yethee version 1.5.1 of @griffel/eslint-plugin works.

**Comment by layershifter** [comment]
Hey folks, thx for reporting! Fixed in `1.6.2`.
